code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -36,34 +37,59 @@
 	var/from_suit = /obj/item/clothing/suit/space/void
 	var/to_helmet = /obj/item/clothing/head/cardborg
 	var/to_suit = /obj/item/clothing/suit/cardborg
+	var/owner_ckey = null //YW ADD
+	
 	//Conversion proc
 /obj/item/device/modkit_conversion/afterattack(obj/O, mob/user as mob)
-	var/flag
+	var/cost
 	var/to_type
-	if(istype(O,from_helmet))
-		flag = 1
+	var/keycheck
+	//we have to check that it's not the original type first, because otherwise it'll convert wrong because the subtype still counts as the basetype
+	//changing an item back to its base type refunds the parts cost
+	if(istype(O,to_helmet))
+		cost = -1
+		to_type = from_helmet
+	else if(istype(O,to_suit))
+		cost = -2
+		to_type = from_suit
+	else if(istype(O,from_helmet))
+		cost = 1
 		to_type = to_helmet
+		keycheck = TRUE
 	else if(istype(O,from_suit))
-		flag = 2
+		cost = 2
 		to_type = to_suit
+		keycheck = TRUE
```

Comment:
```suggestion
		keycheck = TRUE //YW ADD
```

You get the idea more edit comments seeing as we are still editing as VR file.

Unless  you plan on pushing it upstream.